salesagility/SuiteCRM
# Fixed #4035 - Upgrade on 7.9.4 Content in emails are same in body for…

## Description



Fixed a regression issue that was causing the email body to be cached.

## Motivation and Context

Issue reference: #4035 

## How To Test This

1. Repair and rebuild.
2. Read an emails html.
3. Check a second email and see if the body html is different. 

## Types of changes


### Final checklist
